moveit_core/collision_detection/include/moveit/collision_detection/test_collision_common_pr2.h

```diff
@@ -87,6 +87,12 @@ class CollisionDetectorTest : public ::testing::Test
   std::string kinect_dae_resource_;
 };
 
+#ifdef NDEBUG  // Don't perform timing tests in Debug mode (but evaluate expression)
```

Comment:
```suggestion
#ifndef NDEBUG  // Don't perform timing tests in Debug mode (but evaluate expression)
```
Or do I misunderstand this expression?

Reply:
You are right. In my [original commit](https://github.com/ros-planning/moveit/commit/3ebeb650e4cdf0988cde8ae2fd666a2719ea8183) it was the correct expression. Here, it is inverted.

Reply:
You are right. In my [original commit](https://github.com/ros-planning/moveit/commit/3ebeb650e4cdf0988cde8ae2fd666a2719ea8183), it was the correct expression. Here, it is inverted.